[3/3][Inductor] Make CK work in FBCode

ColinPeppler opened the pull request:
Summary:
# Context
Goal: Enable CK for Inductor in FBCode

We split this stack into three diffs to help with review & in case we need to revert anything.

# This Diff
* Gets us to have CK kernels as an option for GEMM autotuning in Inductor.

Reviewed By: zjing14

Differential Revision: D62662705


cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @amjames @desertfire @chauhang

ColinPeppler commented:
Running lintrunner to fix linter issue.

tenpercent reviewed (approved):
Looks good; please wait for the CK backend tests before landing

ColinPeppler committed:
[3/3][Inductor] Make CK work in FBCode (#136234)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/136234

# Context
Goal: Enable CK for Inductor in FBCode

We split this stack into three diffs to help with review & in case we need to revert anything.

# This Diff
* Gets us to have CK kernels as an option for GEMM autotuning in Inductor.

Reviewed By: zjing14

Differential Revision: D62662705
